bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/AnnotatedActions.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.automation;
+
+/**
+ * Marker interface for RuleActions
+ *
+ * Every method in the implementation should provide annotations which are used to create the ModuleTypes
+ *
+ * @author Stefan Triller - initial contribution
+ *
+ */
+public interface AnnotatedActions {
+
+    static final String ACTION_THING_UID = "esh.thingActionID";
```

Comment:
Can you provide a documentation for that variable what it is used for?
E.g. this variable is used for a key of the OSGi service properties, ... or whatever